doc/io.rst

```diff
@@ -47,7 +47,7 @@ read/write netCDF V4 files and use the compression options described below).
 __ https://github.com/Unidata/netcdf4-python
 
 We can save a Dataset to disk using the
-:py:meth:`~Dataset.to_netcdf` method:
+:py:meth:`~xarray.Dataset.to_netcdf` method:
```

Comment:
I think this and the remaining "xarray" changes can be avoided by adding `.. currentmodule:: xarray` at the top of the file. We should just add this at the top of every .rst file I think